src/components/Button.tsx

```diff
@@ -92,7 +131,28 @@ const getStyledButton = (
 		);
 	}
 
-	if ((outline && !active) || !isPrimary) {
+	const themeColorVariants = Object.keys(originalProps.theme.colors);
+	const hasColorVariant = some(themeColorVariants, partial(has, originalProps));
```

Comment:
Can't we just call `getColor` and see if any color was returned?

Reply:
Great point @sradevski - I overcomplicated it there